.github/labeller.yml

```diff
@@ -1,9 +1,13 @@
 labels:
+  - name: "ci"
+    allFilesIn: "(dockerfiles\/ci\/.*|.circleci\/.*|.github\/.*|.gitlab\/.*|benchmark\/.*|github-actions-helpers\/.*|docker-compose.yml)"
+    stopIfMatch: true
+
   - name: "area:asm"
     title: "^\\[?(?i)ASM"
 
   - name: "area:asm"
-    allFilesIn: "(appsec\/.*|docker-compose.yml|.*\/.circleci\/.*)"
+    allFilesIn: "(appsec\/.*|docker-compose.yml|.circleci\/.*)"
```

Comment:
Why do we also match .circleci here (and below) if it's already stopIfMatch for ci?